src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,65 +11,42 @@ redirects:
 
 ## Requirements
 
-If you see this message in the **Version trends** UI: "No adoption data is available for this version because it is running an older agent," this indicates the mobile app wasn't using the New Relic mobile agent version that collects install/upgrade count data.
-
 To collect version trend information for your mobile app, make sure your agent version is:
 
-* New Relic mobile agent for [Android version 5.3.3](/docs/release-notes/mobile-release-notes/android-release-notes/) or higher
-* New Relic mobile agent for [iOS version 5.3.4](/docs/release-notes/mobile-release-notes/ios-release-notes/) or higher
-
-## View version trend data in UI [#view-version-trends]
-
-To view the **Version trends** report:
+* New Relic mobile agent for [Android agent v6.5.0](/docs/release-notes/mobile-release-notes/android-release-notes/) or higher
+* New Relic mobile agent for [iOS agent v7.0.0](/docs/release-notes/mobile-release-notes/ios-release-notes/) or higher
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
-2. Select the **Launch** time period to compare versions by day (default), week, or month, or view all available data.
+## View version trends [#view-version-trends]
 
-   ![Version Trends iOS 2016-01-07 Launch buttons.png](./images/Version-Trends-iOS-2016-01-07-Launch-buttons_0.png "Version Trends iOS 2016-01-07 Launch buttons.png")
-3. Review the [version trend details](#details) to analyze performance, user adoption, and key technical indicators. For additional suggestions about how to interpret the data, review the [examples](#examples).
-4. Optional: To configure email settings, see [Email reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications#version-trends).
+To view the version trends report go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
 
-## View version trend details [#details]
+![This is a screenshot of the version trends UI](./images/version-trends-ui.png "Mobile version trends UI")
 
-Each version includes several metrics to analyze performance, user adoption, and key technical indicators.
-
-<CollapserGroup>
-  <Collapser
-    id="performance"
-    title="Performance across releases"
-  >
-    An important feature is comparison of app performance across releases, based on data collected when each version was at its peak of popularity. You can select four different views of data for when each version was in production, including:
+Each new version of your mobile app is compared against the previous version. For a selected time period, the version trends report compares the performance increases and decreases of your app versions against each other.
```

Comment:
These two sentences seem to be contradicting each other. But maybe I'm misreading it?

The first sentence seems to be saying that each version performance is compared to the immediately previous version. But the second sentence seems to say that all of the app versions are compared against each other. 

Is the behavior different for a selected time period?